docs/section10/10-7.md

```diff
@@ -0,0 +1,69 @@
+                    - Check else if capacity is <240,000 Btuh and greater or equal to 135,000 Btuh: `elif total_cool_capacity_b < 240000 AND total_cool_capacity_b >= 135000: size_category_b = 135_240Btuh`
+                    - Check else if capacity is <760,000 Btuh and greater or equal to 240,000 Btuh: `elif total_cool_capacity_b < 760000 AND total_cool_capacity_b >= 240000: size_category_b = 240_760Btuh`
+                    - Else capacity is >=760,000 Btuh: `Else: size_category_b = 760Btuh`
+                    - Set the COPnf_full_load_b variable using data_lookup: `COPnf_full_load_b = data_lookup(table_G3_5_1,size_category_b)`
+                - Else, system type is 4:`Else:` 
+                    - Check if capacity is <65,000 Btuh: `if total_cool_capacity_b < 65000: size_category_b = 65Btuh`
+                    - Check else if capacity is <135,000 Btuh and greater or equal to 65,000 Btuh: `elif total_cool_capacity_b < 135000 AND total_cool_capacity_b >= 65000: size_category_b = 65_135Btuh`
+                    - Check else if capacity is <240,000 Btuh and greater or equal to 135,000 Btuh: `elif total_cool_capacity_b < 240000 AND total_cool_capacity_b >= 135000: size_category_b = 135_240Btuh`
+                    - Else capacity is >=240,000 Btuh: `Else: size_category_b = 240Btuh`
+                    - Set the COPnf_full_load_b variable using data_lookup: `COPnf_full_load_b = data_lookup(table_G3_5_2,size_category_b)`
+                This part directly below is because mismatches between expected and modeled cooling efficiency could be due to there being multiple hvac zones combined into a single thermal block for specific system types.
+                - Check if system type is 3 or 4: `if hvac_sys_type_b in ["SYS-3", "SYS-3b", "SYS-4]:` 
+                    - Set the system type 3 or 4 boolean variable to TRUE: `sys_type_3_or_4 = TRUE`      
+                    - Check if the capacity bracket is the smallest: `if size_category_b == 65Btuh: smallcapBracket = TRUE`  
+                                                
+                **Rule Assertion:**
+                - Case 1: if COPnf_full_load_b equals full_load_efficiency_b and the system is not system type 3 or 4 then pass: `if COPnf_full_load_b == full_load_efficiency_b and sys_type_3_or_4 == FALSE: PASS`
+                - Case 2: else if COPnf_full_load_b equals full_load_efficiency_b and the system is system type 3 or 4 and is in the smallest capacity bracket then pass: `elif COPnf_full_load_b == full_load_efficiency_b and sys_type_3_or_4 == TRUE and smallcapBracket == TRUE: PASS`           
+                - Case 3: else if COPnf_full_load_b does not equal the full_load_efficiency_b and the system is system type 3 or 4 and is in the smallest capacity bracket then pass: `elif COPnf_full_load_b != full_load_efficiency_b and sys_type_3_or_4 == TRUE and smallcapBracket == TRUE: FAIL`            
```

Comment:
Description should be changed from pass to fail, to match the pseudocode.

Reply:
Done